pyteal/ast/abi/address.py

```diff
@@ -22,7 +22,7 @@ class AddressLength(IntEnum):
 
 class AddressTypeSpec(StaticArrayTypeSpec):
     def __init__(self) -> None:
-        super().__init__(ByteTypeSpec(), AddressLength.Bytes)
+        super().__init__(ByteTypeSpec(), int(AddressLength.Bytes))
```

Comment:
abi-round-trip test exposed this issue. Is there a better way to handle this?

Reply:
@tzaffi I'm not sure that it's _better_, but for comparison purposes, I opened a PR based on local changes:  https://github.com/algorand/pyteal/pull/347.

Appreciate a review as you have time.  Feel welcomed to close it if you prefer as is.

Reply:
merged in